python/lsst/afw/cameraGeom/_camera.py

```diff
@@ -21,10 +21,10 @@
 
 # Camera object below is the same one exported by the pybind11 camera
 # module, so we don't need to re-export it here.
-__all__ = []
+__all__ = ['Camera']
```

Comment:
This change looks inconsistent with the comment above it.  If we need the change, we should remove or modify the comment.